Frontend/ui-customer/src/Flow.purs

```diff
@@ -87,6 +87,10 @@ baseAppFlow gPayload = do
   _ <- pure $ setValueToLocalStore TEST_POLLING_INTERVAL "1500.0"
   _ <- pure $ setValueToLocalStore TEST_POLLING_COUNT "113"
   _ <- pure $ setValueToLocalStore RATING_SKIPPED "false"
+  if ((getValueToLocalStore SESSION_ID == "__failed") || (getValueToLocalStore SESSION_ID == "(null)")) then do
```

Comment:
if you want you can do something like 
`when (condition) $ do
      setValueToLocalStore SESSION_ID (generateSessionId unit)`